pkg/common/k8s_utils.go

```diff
@@ -147,3 +153,18 @@ func GetLabel(obj metav1.Object, key string) string {
 	}
 	return obj.GetLabels()[key]
 }
+
+func GenerateClusterID(ctx context.Context, c client.Client) (error, string) {
+	//Already calculated? return it
+	if clusterID != "" {
+		return nil, clusterID
+	}
+
+	ns := &corev1.Namespace{}
+	err := c.Get(ctx, client.ObjectKey{Name: clusterIDNamespace}, ns)
+	if err != nil {
+		return err, ""
+	}
+	clusterID = ToBase36HashLen(string(ns.UID), ClusterIDLength)
+	return nil, clusterID
```

Comment:
Why can't this not just return string(ns.UID) and the decision on hashing it or not can be done eslewhere?

Reply:
yes ok, this makes sense, I'll do it that way